sagemaker-script-mode/pytorch-sagemaker-huggingface/huggingface_text_classification.ipynb

```diff
@@ -0,0 +1,1171 @@
+{
```

Comment:
<p>lets get rid of this output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>